app/transformers/api/current/facility_transformer.rb

```diff
@@ -2,9 +2,9 @@ class Api::Current::FacilityTransformer
   class << self
     def to_response(facility)
       facility.as_json
-        .except('enable_diabetes_management')
+        .except('enable_diabetes_management', 'monthly_opd_load')
```

Comment:
I believe extra fields in the API response are gracefully ignored by the app. You can confirm with the mobile devs and remove this change if so.

Reply:
The idea here is to not let users access this field, since this is admin-only and we should maybe exclude it in the API response.

Reply:
Herpderp, makes perfect sense.